docs/_static/network/v1.yaml

```diff
@@ -101,7 +101,6 @@ paths:
           content:
             application/json:
               schema:
-                type: object
```

Comment:
must not be removed (actual a conflict with master)

Reply:
Reverted it. Didn't actually change code since we override that type with our own, since the code generator doesn't properly handle additionalProperties (making it impossible to consume/produce a map)